### ISSUE TITLE
Events End Date Not Exporting column data.

### ISSUE THREAD

**Issue by webfalter** [artistro08/tailor-starter#9]
October Build 3.0.72
PHP 8.1

https://github.com/artistro08/tailor-starter/blob/main/seeds/blueprints/content/events/event.yaml

When exporting (json) are not saved date /  end_date

**Comment by webfalter** [comment]
Hallo @artistro08 
Thank you, I could already guess
At the moment I have other construction sites, so the export can wait a little longer.
I'll create a ticket for it at some point

**Comment by artistro08** [comment]
@webfalter Just a heads up v3.1 Fixes the export issue. Tested myself and it works flawlessly

**Comment by webfalter** [comment]
@artistro08 
I still don't get an export of date / end_date
An error message comes up sometimes, mostly everything runs normally

**Comment by artistro08** [comment]
Try clearing your cache before exporting. not sure why it's not exporting that. Also make sure it's one of the checked items as well.

**Comment by webfalter** [comment]
The elements for possible marking do not even appear for me in the date reference.
I have to say that I'm not using your code, I've just looked at your good example of how to use Tailor without reading the docs first
I'll install your complete code one day and see why it doesn't work for me.

Thank you for your feedback

**Comment by artistro08** [comment]
I'll investigate

**Comment by webfalter** [comment]
I can't find the right place in the CMS Docs, but it seems to work with the extension.      
```
 column:
            type: summary
```
![Bildschirmfoto 2022-10-17 um 11 19 35](https://user-images.githubusercontent.com/481551/196141024-14d75c26-7146-4023-831b-b5cb5e0d3554.png)

**Comment by artistro08** [comment]
That would make sense. I'll update it when I have some time. In the meantime, I'll refer people to this fix

**Comment by artistro08** [comment]
Another fix for this would be setting the column to invisible like so. This is the fix I'll implement when I get time. Example below

```yaml
column:
  invisible: true
```

**Comment by webfalter** [comment]
Probably everything helps with set column
Thanks to Tailor, you can certainly tackle other exciting projects

**Comment by artistro08** [comment]
fixed in v3.1.0
